api/src/routes/datasetUploads.js

```diff
@@ -84,6 +85,61 @@ router.get(
   }),
 );
 
+// router.get(
+//     '/:username/all',
+//     validate([
+//       query('status').isIn(Object.values(config.upload.status)).optional(),
+//       query('dataset_name').notEmpty().escape().optional(),
+//       query('limit').isInt({min: 1}).toInt().optional(),
+//       query('offset').isInt({min: 0}).toInt().optional(),
+//       param('username').escape().notEmpty()
+//     ]),
+//     isPermittedTo('read', {checkOwnerShip: true}),
+//     asyncHandler(async (req, res, next) => {
+//       // #swagger.tags = ['uploads']
+//       // #swagger.summary = 'Retrieve past uploads'
+//
+//       const {
+//         status, dataset_name, offset, limit,
+//       } = req.query;
+//
+//       const query_obj = {
+//         where: _.omitBy(_.isUndefined)({
+//           upload_log: {
+//             status,
+//             // user_id: req.params.id,
+//             user: {
+//               username: req.params.username,
+//             },
+//           },
+//           dataset: {
+//             name: {contains: dataset_name},
+//           },
+//         }),
+//       };
+//       const filter_query = {
+//         skip: offset,
+//         take: limit,
+//         ...query_obj,
+//         orderBy: {
+//           upload_log: {
+//             initiated_at: 'desc',
+//           },
+//         },
+//         include: INCLUDE_DATASET_UPLOAD_LOG_RELATIONS,
+//       };
+//
+//       const [dataset_upload_logs, count] = await prisma.$transaction([
+//         prisma.dataset_upload_log.findMany({
+//           ...filter_query,
+//         }),
+//         prisma.dataset_upload_log.count({...query_obj}),
+//       ]);
+//
+//       res.json({metadata: {count}, uploads: dataset_upload_logs});
+//     }),
+// );
+
```

Comment:
No issues with keeping this in here for now. This will be needed in an upcoming feature (open Uploads to role User)